sale_mrp_bom/security/security.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="sale_mrp_bom_group" model="res.groups">
+        <field name="name">Allows definie BOM on sale order line</field>
```

Comment:
```suggestion
        <field name="name">Allows to define a BOM on sale order line</field>
```
Sounds better

Reply:
@HaraldPanten thanks for your review, Done!